sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -118,6 +118,14 @@ var doc = JsonDocument.Parse(response.Content.ToMemory());
 var name = doc.RootElement.GetProperty("name").GetString();
 ```
 
+JSON properties can also be accessed using a dynamic layer.
```

Comment:
I used a new fake WidgetClient instead of the test PetStoreClient because the other is used in a lot of tests and I wanted something specifically for samples. If we don't like this approach, I can look at either moving this article to the WidgetClient completely, or better understanding the needs of the PetStoreClient.  Both are generated from fake swagger files.

Reply:
Should we be using something like `CattleClient` and other bovine-related properties as examples so that we stay on-brand for our current initiatives?

_( yes, this is said in jest.  😄 )_

Reply:
🐮

Reply:
Now those jokes will be lost to the ether. 😥